homeassistant/components/media_player/services.yaml

```diff
@@ -322,3 +322,40 @@ epson_select_cmode:
     cmode:
       description: Name of Cmode
       example: 'cinema'
+
+soundtouch_play_everywhere:
```

Comment:
We don't want to add the descriptions here. We want to move the registration of the services from the media_player domain to the soundtouch domain.

Reply:
I understand your point, but isn't this the same as the `neato` integration as stated in the [issue](https://github.com/home-assistant/home-assistant/issues/27289):

> (example neato integration registers a specific service for the platform vacuum and the service documentation is in homeasssitant/components/vacuum/services.yaml

The soundtouch integration registers its services for the platform media_player, e.g:
https://github.com/home-assistant/home-assistant/blob/de4482e8d3cc2659d7f4898bdeaa6bf1b010e663/homeassistant/components/soundtouch/media_player.py#L154-L159

So I thought I would follow the `neato` example and add the services to media_player/services.yaml. Or is there any important reason that I am missing why it is better to move the registration of the services to the soundtouch domain? Sorry for the trouble I just want to fully understand this matter :)

Reply:
> If you can't test your change and if you don't want to write unit tests I don't think we should continue here. Testing is required before we can accept a PR.

Please do not get me wrong, I have no problem writing unit test and testing my code for PRs at all, but I thought that changing services.yaml files did not require any unit testing.

Since I started this PR the following lines have been added to the issue:
> In consequence, please update only services documentation for integration you can test with you home assistant install. Ideally provide a screenshot in your PR as proof of your validation.


I would not have made this PR if I had known that I had to locally have in this case a soundtouch device and so on.

I completely understand that testing is necessary and important before accepting a PR.
Hopefully someone can pick up my code and use it to finish the services.yaml for these integrations.

Reply:
The neato example in the issue is not a good example since we don't want that behavior in the first place.

https://developers.home-assistant.io/docs/en/dev_101_services.html

Reply:
Oh I am so sorry, I completely read that wrong. My bad.